tensorflow_serving/model_servers/server.cc

```diff
@@ -263,6 +263,14 @@ Status Server::BuildAndStart(const Options& server_options) {
   builder.AddListeningPort(
       server_address,
       BuildServerCredentialsFromSSLConfigFile(server_options.ssl_config_file));
+  // If defined, listen to a UNIX socket for gRPC.
+  const string grpc_socket_path = server_options.grpc_socket_path;
+  const string grpc_socket_uri = "unix:" + grpc_socket_path;
+  if (!grpc_socket_path.empty()) {
```

Comment:
suggestion: May want to directly do if (server_options.grpc_socket_path.empty()) ?

Reply:
Yes, good call! I went with that since it aligned so well with previous code on that file.